<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="0">

  <testsuite name="rewardsTest"
    errors="1" failures="0" hostname="" id="" package="rewardsTest" skipped="1"
    tests="0" time="0.001000" timestamp="">
  

  
    <system-err>
       An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if &#34;start_process&#34; is set to &#34;true&#34;.
    </system-err>
  

    
    <error message="An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if &#34;start_process&#34; is set to &#34;true&#34;."><![CDATA[
      Error: An error occurred while creating a new GeckoDriver session: Connection refused to localhost:4444. If the Webdriver/Selenium service is managed by Nightwatch, check if &#34;start_process&#34; is set to &#34;true&#34;.
    at Timeout._onTimeout (D:\nightwatch-pt\node_modules\nightwatch\lib\transport\selenium-webdriver\httpclient.js:90:27)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7)
      ]]></error>  

  
    
    <testcase
      name="rewards page check" classname="rewardsTest">
      <skipped />
    </testcase>
    
  
  </testsuite>
</testsuites>
